### PR TITLE
Add way to run all examples

### PR DESCRIPTION
**Pull request by stloyd**
<h2>Change Log</h2>
<div id="change-log">
  <h4>Added</h4>
  <ul id="added">
    <li>Add way to run all examples</li>
  </ul> 
  <h4>Fixed</h4>  
  <ul id="fixed">
    
  </ul>
  <h4>Changed</h4>
  <ul id="changed">
    
  </ul>  
  <h4>Removed</h4>
  <ul id="removed">
    
  </ul>
  <h4>Deprecated</h4>
  <ul id="deprecated">
    
  </ul>  
  <h4>Security</h4> 
  <ul id="security">
    
  </ul>     
</div>
<hr/>

<h2>Description</h2>

```console
$ examples/run_examples.php
Running all available examples.

Example: dataframe/get.php
array(2) {
  ["id"]=>
  int(1)
  ["name"]=>
  string(3) "foo"
}

...
```

### CODE REVIEW

**Comment by norberttech** [comment]
Could you try to run it with a clean `/examples/output` folder? I think it might fail there

**Comment by stloyd** [comment]
In case of failure in example, the output will be like:
```console
Example: types/csv/csv_to_parquet_10k.php
Converting CSV 89Mb file into parquet...
Example failed: Destination path "file://Users/stloyd/Documents/flow/examples/output/dataset_10k.parquet" already exists, please change path to different or set different SaveMode

Example: types/csv/csv_to_avro.php
Converting CSV 89Mb file into avro...
Example failed: Destination path "file://Users/stloyd/Documents/flow/examples/output/dataset.avro" already exists, please change path to different or set different SaveMode
```

**Comment by stloyd** [comment]
Reworked the script to run cleanup & setup tasks as well, and moved one setup scripts to the upper-level directory.

File: examples/topics/types/csv/csv_to_parquet_100k.php [hunk 1]
```diff
@@ -11,6 +11,12 @@
 
 require __DIR__ . '/../../../bootstrap.php';
 
+if (!\file_exists(__FLOW_OUTPUT__ . '/dataset.csv')) {
+    print "Data file is missing, please first run: php examples/setup/php_to_csv_and_json.php\n";
```
Comment (norberttech) [inline examples/topics/types/csv/csv_to_parquet_100k.php]: in that case, can we just run the other example called `php_to_csv_and_jsonphp` ?

Comment (stloyd) [reply examples/topics/types/csv/csv_to_parquet_100k.php]: Done.

**Pull request merged**